src/rleCompress.js

```diff
@@ -18,7 +18,21 @@
  * @return {string}
  */
 function rleCompress(source) {
-  // write code here
+  if (!source || source === '') {
```

Comment:
пустая строка в булевом значении `false`
<img width="166" alt="Screen Shot 2019-08-19 at 10 59 34 AM" src="https://user-images.githubusercontent.com/36257947/63248638-72d1f980-c270-11e9-943b-4ffa26ff5d41.png">
```suggestion
  if (!source) {
```